spyder/plugins/editor.py

```diff
@@ -1977,6 +1990,20 @@ def unindent(self):
             editor.unindent()
 
     @Slot()
+    def text_uppercase (self):
+        """Change current line or selection to uppercase"""

```

Comment:
Add a . in the end.

Docstrings are sentences

`"""Change current line or selection to uppercase."""`